src/pages/api/logs.ts

```diff
@@ -0,0 +1,24 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const referer = req.headers.referer;
+
+  const allowedReferers = ['https://www.moregoats.com/', 'https://dev.moregoats.com/'];
+
+  if (!referer || !allowedReferers.includes(referer)) {
+    res.status(403).json({ error: 'Unauthorized access' });
+    return;
+  }
```

Comment:
This is a nice to have, but I don't think it is necessary for this project. It's just a log, if we can tons we can create a filter. I don't think it will happen tho

Reply:
removed [3e2d271](https://github.com/defi-wonderland/world-id-goats-dao-ui/pull/24/commits/3e2d2718c38783f3c8b398f2577e2892c386e5cc)